cmd/dep/testdata/harness_tests/init/govendor/case1/final/Gopkg.toml

```diff
@@ -0,0 +1,9 @@
+ignored = ["github.com/sdboyer/dep-test"]
+
+[[constraint]]
+  name = "github.com/carolynvs/go-dep-test"
```

Comment:
In the `case1` vendor file, the ignore list contains three items, but only one is valid.
- `test` isn't a package identifier, so it's skipped
- `github.com/sdboyer/dep-test` is a valid package so it's added to the ignore list
- `samples/` is a package prefix, which isn't supported by dep, so we skip it

If `samples/` was full path I think you'd be right. Please let me know if you think I've misunderstood the govendor ignore directive.